<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gb_operation">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Operation</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_run">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_baud">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>211</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_saveGraph">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Graph</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Baudrate</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_port">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>301</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clear">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_connect">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_refresh">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_horizontal">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Horizontal axis</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of samples:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_numSamples">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::DefaultStepType</enum>
     </property>
     <property name="value">
      <number>10000</number>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rbtn_xFixRange">
        <property name="text">
         <string>Fix range</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtn_xAutoScale">
        <property name="text">
         <string>Auto Scale</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_vertical">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Vertical axis</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Min:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_min">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_max">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="value">
      <double>3.300000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Min:</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>32</y>
       <width>157</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbtn_yFixRange">
        <property name="text">
         <string>Fix range</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtn_yAutoScale">
        <property name="text">
         <string>Auto scale</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_graph">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>1221</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Graph</string>
    </property>
    <widget class="QCustomPlot" name="graph" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1201</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_dataScale">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>9</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Data scale</string>
    </property>
    <widget class="QCheckBox" name="chb_scaleEnable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale enable</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Raw range:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Scaled range:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_minRaw">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_maxRaw">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_minScale">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_maxScale">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>to</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>to</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
